### ISSUE TITLE
Grammar allows for whitespace-only scope

### ISSUE THREAD

**Issue by epage** [conventional-commits/parser#39]
I'm assuming this is not intended but I'm unsure if its worth specifying this out of scope.

**Comment by wesleytodd** [comment]
I don't think it was intended and I don't recall any conversation around that. Also, looking at the implementation this is not considered an error: https://github.com/conventional-commits/parser/blob/main/lib/parser.js#L183

I will mark this has a help wanted in case someone wants to open a PR to update the grammar and implement it in the parser.
